library/alloc/src/raw_vec.rs

```diff
@@ -1,5 +1,4 @@
-#![unstable(feature = "raw_vec_internals", reason = "implementation detail", issue = "none")]
-#![doc(hidden)]
+#![unstable(feature = "raw_vec_internals", reason = "unstable const warnings", issue = "none")]
```

Comment:
Can this be removed since it is no longer exported?

Reply:
I tried that but got

```
error: const-stable function cannot use `#[feature(const_fn_trait_bound)]`
   --> library/alloc/src/raw_vec.rs:104:9
    |
104 | impl<T, A: Allocator> RawVec<T, A> {
    |         ^
    |
help: if it is not part of the public API, make this function unstably const
    |
121 |     #[rustc_const_unstable(feature = "...", issue = "...")]
    |
help: otherwise `#[rustc_allow_const_fn_unstable]` can be used to bypass stability checks
    |
121 |     #[rustc_allow_const_fn_unstable(const_fn_trait_bound)]
    |

error: `Unique::<T>::dangling` is not yet stable as a const fn
   --> library/alloc/src/raw_vec.rs:123:21
    |
123 |         Self { ptr: Unique::dangling(), cap: 0, alloc }
    |                     ^^^^^^^^^^^^^^^^^^
    |
    = help: const-stable functions can only call other const-stable functions
```

Adding `rustc_allow_const_fn_unstable` solves the former but not the latter.